### ISSUE TITLE
RTP Nether

### ISSUE THREAD

**Issue by Squidistry** [WiIIiam278/HuskHomes#650]
RTP nether can never find a safe place to put players. Is it not supported yet for the nether?

**Comment by WiIIiam278** [comment]
Yeah this is intended as /rtp in the nether would need a different safety logic check to account for the terrain
